### ISSUE TITLE
T-Mobile

### ISSUE THREAD

**Issue by pabogan** [GrapheneOS/grapheneos.org#712]
Yesterday I switched from AT&T to T-mobile on my Pixel 6 pro running latest graphene OS.  Now I am not able to receive data on the phone.  I can receive calls and text messages, just no data.  Open Vanadium and type in Google, it times out and tells me to check the connection.  All other searches do the same.  Before using AT&T I had no issues with data on the phone.

**Comment by thestinger** [comment]
You need to fix your APN configuration. Start by resetting network configuration.

**Comment by thestinger** [comment]
Please ask for support on the discussion forum if that doesn't resolve it. It's not a bug.

https://discuss.grapheneos.org/

**Comment by pabogan** [comment]
Well I checked around 1 pm today and all of a sudden I started receiving emails and was able to
use internet for searches. Not sure if their engineering people did something. It was elevated to
them after the support staff could not correct the problem.

When I tried to locate the APN configuration for my phone I see T-Mobile US, fast.t-mobile.com. When
I tap on that to go to Edit access point everything is greyed out. I am not able to change anything!

I can see the following:
Proxy not set
Port not set
Username not set
Password not set
Server not set
MMSC http://mms.msg.eng.t-mobile.com/mms/wapenc
MMS proxy not set
MMS port not set
MCC 310
MNC 260
Authentication type not set
APN type default,supl,ia,mms,xcap
APN protocol IPv6
APN roaming protocol IPv4
APN enable/disable APN enabled
Bearer HSPA, EVDO_B, eHRPD, LTE, HSPAP, GPRS, EDGE, UMTS, IS95A, IS95B, 1xRTT, EVDO_0, EVDO_A, HSDPA, NR, HSUPA
MVNO type None
MVNO value not set

THEN WHEN I CHECKED THE PHONE AROUND 4 PM, BACK TO NO INTERNET SERVICE. NOT OS UPDATES, NO EMAILS.

How do I access the APN settings if they are greyed out?

Paul
